Vagrant/.vagrant/machines/master/virtualbox/action_provision

```diff
@@ -0,0 +1 @@
+1.5:4d28ffe5-0e15-49f7-9577-e3516e389471
```

Comment:
The whole .vagrant directory contains your local vagrant state and it is not intended to be here in github. 
Remove .vagrant dir and just keep `Vagrantfile ` here